### PR TITLE
Issue #21: Added contract creation details to contract summary

### LINKED ISSUE #21: Add contract type and creation date to contract summary view
Depends on cibr-qcri/airflow-tasks#42

### PR DESCRIPTION
**Pull request by thedarkspy**
Closes #21

### CODE REVIEW

**Comment by thedarkspy** [comment]
I suggest adding contract type at a later stage when we decide what to label ERC20 and ERC721 contracts. Currently true or false values do not add much to the context. Moreover, since they are not always available, this might miss the even number of summary fields we have. We can also add them as checkboxes that are ticked whenever a value is true.

**Comment by thedarkspy** [comment]
Screen Shot 2022-09-04 at 9 39 02 AM

**Comment by thedarkspy** [comment]
Screen Shot 2022-09-04 at 11 00 32 AM

Updated UI

File: api/controllers/contracts.js [hunk 1]
```diff
@@ -13,11 +14,13 @@ const getContract = asyncHandler(async (request, response, next) => {
 
   const result = await gp.query(dbQueries.getContractByAddress, [address]);
 
+  console.log(result.rows);
```
Comment (boshmaf) [inline api/controllers/contracts.js:17]: Please remove

File: api/utils/contracts.js [hunk 2]
```diff
@@ -1,6 +1,34 @@
 const numeral = require('numeral');
 const { timestampToDateString } = require('./common');
 
+exports.transformContractInfo = (row) => {
+  return {
+    id: row.id,
```
Comment (boshmaf) [inline api/utils/contracts.js:6]: Use JS case styles for naming: camelCase

File: web/src/components/Contract/Summary/Summary.js [hunk 3]
```diff
@@ -12,12 +12,12 @@ import { showTagDialog } from '../../../store/actions';
 const fieldToLabelMap = {
   name: 'Name',
   symbol: 'Symbol',
-  address: 'Address',
   security_score: 'Security Score',
-  total_spent: 'Amount Sent',
-  total_received: 'Amount Received',
-  num_tx_spent: 'Transactions Sent',
-  num_tx_received: 'Transactions Received',
+  total_received: 'Total In',
```
Comment (boshmaf) [inline web/src/components/Contract/Summary/Summary.js:16]: Please use camelCase for names

**Review by boshmaf — CHANGES_REQUESTED** [review]

**Review by boshmaf — APPROVED** [review]

**Pull request merged**